pandas/core/generic.py

```diff
@@ -1109,10 +1109,20 @@ def to_json(self, path_or_buf=None, orient=None, date_format='epoch',
               - index : dict like {index -> {column -> value}}
               - columns : dict like {column -> {index -> value}}
               - values : just the values array
+            - ``'table_schema'`` dict like
+              ``{'schema': [schema], 'data': [data]}``
+              where the schema component comes from
+              ``build_table_schema`` and the data component is
+              like ``orient='records'``.
+
+            .. versionchanged:: 0.20.0
+            Added the ``'table_schema'`` orient.
```

Comment:
you don't need the ``Add the ...`` line, instead indent the versionchanged (I think), @jorisvandenbossche